serverless/src/getAccessMethods/handler.js

```diff
@@ -0,0 +1,70 @@
+import { getValueForTag, hasTag } from '../../../sharedUtils/tags'
+import { getOptionDefinitions } from './getOptionDefinitions'
+
+import { getJwtToken } from '../util'
+
+const getAccessMethods = async (event) => {
+  const { body } = event
+  const { params = {} } = JSON.parse(body)
+
+  // TODO will need collectionId for default access configurations
+  // eslint-disable-next-line no-unused-vars
+  const { collection_id: collectionId, tags } = params
+
+  const jwtToken = getJwtToken(event)
+
+  const hasEchoOrders = hasTag({ tags }, 'subset_service.echo_orders')
+  const hasEsi = hasTag({ tags }, 'subset_service.esi')
+  const hasOpendap = hasTag({ tags }, 'subset_service.opendap')
+  const capabilitiesData = getValueForTag('collection_capabilities', tags)
+  const { granule_online_access_flag: downloadable } = capabilitiesData || {}
+
+  const accessMethods = {}
+  if (downloadable) {
+    accessMethods.download = {
+      isValid: true,
+      type: 'download'
+    }
+  }
+
+  if (hasEchoOrders) {
+    const echoOrderData = getValueForTag('subset_service.echo_orders', tags)
+    const { option_definitions: optionDefinitions } = echoOrderData
+
+    if (optionDefinitions) {
+      const forms = await getOptionDefinitions(optionDefinitions, jwtToken)
+
+      forms.forEach((form) => {
+        const [key] = Object.keys(form)
+        accessMethods[key] = {
+          ...echoOrderData,
+          ...form[key],
+          type: 'ECHO_ORDER'
```

Comment:
This shouldn't be hard coded, it should be the `echoOrderData.Type`, it should contain the `Type` from UMM S which in this case is `ECHO ORDERS` and in the other two will be `ESI` and `OPeNDAP` (See [here](https://github.com/nasa/Common-Metadata-Repository/blob/ea2c519d9a3647c1952da02c95d9a93f731ec7a2/umm-spec-lib/resources/json-schemas/service/umm/v1.2/umm-s-json-schema.json#L342))